### PR TITLE
Feature/censorship methods

### PR DESCRIPTION
**Pull request by hellais**
In here I added support for viewing the censorship methods inside of the country level view. I have also imported the known censorship methods in the production DB.

### CODE REVIEW

File: client/ngapp/build/main.js [hunk 1]
```diff
@@ -66,6 +65,15 @@ angular.module('ooniAPIApp')
         console.log('error', error)
       })
 
+    Country.findOne({
```
Comment (simonv3) [inline client/ngapp/build/main.js]: I'm getting `Country is not defined` here, probably needs to be imported in the controller?

File: client/ngapp/scripts/controllers/country.js [hunk 2]
```diff
@@ -24,6 +23,15 @@ angular.module('ooniAPIApp')
         console.log('error', error)
       })
 
+    Country.findOne({
```
Comment (simonv3) [inline client/ngapp/scripts/controllers/country.js]: Oh hey, actual change above is here, the above comment was on the compiled file. (Should we remove the compiled file form the repo by the way?)

File: client/ngapp/build/main.js (hunk 1 above)
Comment (hellais) [inline client/ngapp/build/main.js]: Yeah I think it's probably due to something having messed up in rebasing the branch. Let me fix this.

File: client/ngapp/scripts/controllers/country.js (hunk 2 above)
Comment (hellais) [inline client/ngapp/scripts/controllers/country.js]: Yeah that is probably a good idea. Will do.

**Comment by hellais** [comment]
Ok this should now be all good.

**Comment by simonv3** [comment]
Looks like it works great! Thanks for cleaning up the main.js file :)

**Pull request merged**